docs/visual-basic/misc/bc30112.md

```diff
@@ -1,4 +1,5 @@
 ---
+description: "Learn more about: <typename>' is a namespace and cannot be used as an expression"
```

Comment:
When the first word is surrounded by apostrophes, the first apostrophe is getting dropped from the description text.

Reply:
Sorry, I should have been clearer -- I only marked one instance in the comments but there are a number of them, the same fix needs to be applied to all of them.  Same clarification applies to batch 9.

Reply:
Yup, noticed that after I fixed the one. Just pushed another ~400. I've requested an update to the tool that does this scan. I'm wondering if it would be better to close off these two and re-run once the new version is available? (unless I got them all in this one)

Reply:
>wondering if it would be better to close off these two and re-run once the new version is available? 

That makes sense to me unless it will be a long time before the fix is available.

Reply:
Hopefully Monday, but it might mean re-running from master. I'll see what Gene has to say.

Reply:
Fixed all I could find (both ends) in this PR